packages/backend-core/src/users/utils.ts

```diff
@@ -16,30 +16,55 @@ export const hasAdminPermissions = sdk.users.hasAdminPermissions
 export const hasBuilderPermissions = sdk.users.hasBuilderPermissions
 export const hasAppBuilderPermissions = sdk.users.hasAppBuilderPermissions
 
-export async function isCreator(user?: User | ContextUser) {
+async function getGroups(groupIds: string[]) {
+  if (groupIds.length) {
```

Comment:
This feels like an optimisation we should do at the `getMultiple` level. It's broadly applicable.